### ISSUE TITLE
makemeasandwich says "Logging in" then exits

### ISSUE THREAD

**Issue by stsievert** [travist/makemeasandwich.js#22]
When I run makemeasandwich, it's completely broken. It can't even log in.

``` shell
$ sudo makemeasandwich --debug
Logging in.
$
```

Maybe something changed in what Jimmy Johns returns?

Details:

System: OSX 10.11.

**Comment by travist** [comment]
Yeah... I need to revisit this library and update it. This is totally a weekend warrior project for me so I will someday find the time.  You know how it goes....  ;)

**Comment by stsievert** [comment]
I totally understand. But man, this is a cool project (and better than ordering jimmy johns pointing and clicking through their web interface).
